content/src/content/jcr_root/apps/core/wcm/components/languagenavigation/v1/languagenavigation/README.md

```diff
@@ -15,7 +15,7 @@ limitations under the License.
 -->
 Language Navigation (v1)
 ====
-Language Navigation component written in HTL that renders a global language structure navigation.
+Language Navigation component is written in HTL that renders a global language structure navigation.
```

Comment:
@NinjaCoderDev - the previous description makes more sense than the change here, I would it as it was.